src/components/Map/Map.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {
+  act,
+  fireEvent,
+  render as renderRtl,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { mockMediaQuery } from '@test/testUtils';
+
+import type { MapProps, Location } from './Map';
+import { Map } from './Map';
+
+const mediaQueryBreakPoint = 500;
+const { setScreenWidth } = mockMediaQuery(mediaQueryBreakPoint);
+
+const user = userEvent.setup();
+
+describe('Map', () => {
+  beforeEach(() => {
+    setScreenWidth(mediaQueryBreakPoint + 100);
+  });
+
+  describe('Marker', () => {
+    it('should show marker when marker is set', () => {
+      render({
+        markerLocation: { latitude: 59.2641592, longitude: 10.4036248 },
+      });
+      expect(locationMarker()).toBeInTheDocument();
+    });
+
+    it('should not show marker when marker is not set', () => {
+      render({ markerLocation: undefined });
+      expect(locationMarker()).not.toBeInTheDocument();
+    });
+  });
+
+  describe('Click map', () => {
+    it('should call mapClicked with correct coordinates when map is clicked', async () => {
+      const handleMapClicked = jest.fn();
+      render({
+        onClick: handleMapClicked,
+      });
+
+      clickMap();
```

Comment:
Related to comment on `clickMap` helper function

```suggestion
      await clickMap();
```

Reply:
Fixed